modules/server/src/utils/getIndexPrefix.js

```diff
@@ -0,0 +1,9 @@
+function getIndexPrefix({ projectId, type }) {
```

Comment:
why take the object type here?

Reply:
instead of just index?

Reply:
I updated to just pass index 👍  initially I had thought I would have type available everywhere, but that was wrong. this makes more sense! :)

Reply:
yea it looks like you're only using `index` here, and in some places above passing `type : { index }`. was just wondering if maybe we would use it later